client/src/components/Modals/ClaimPitch/index.tsx

```diff
@@ -224,7 +229,8 @@ const ClaimPitchModal: FC<ClaimPitchProps> = ({
                   disabled={
                     (slot[1].target - slot[1].current <= 0 &&
```

Comment:
Would it be possible to abstract this logic out into a helper function, it seems to be getting a little messy with more and more we are adding